pkg/appStore/deployment/repository/InstalledAppRepository.go

```diff
@@ -731,7 +732,7 @@ func (impl InstalledAppRepositoryImpl) GetInstalledAppByGitHash(gitHash string)
 	return model, nil
 }
 
-func (impl InstalledAppRepositoryImpl) GetInstalledAppByAppId(appId int) (InstalledApps, error) {
+func (impl InstalledAppRepositoryImpl) GetAcdInstalledAppByAppId(appId int) (InstalledApps, error) {
```

Comment:
This function can be GetInstalledAppByAppIdAndDeploymentType

Reply:
fixed 👍